.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ concurrency:
 
 env:
   CI: true
-  dist: ember-toucan-core/dist
+  dist: packages/ember-toucan-core/dist
```

Comment:
Idk if this setup is going to scale with multiple packages? We might consider using `turbo` here as well? (as in headless-form)

Reply:
> Idk if this setup is going to scale with multiple packages?

Yeah I think you're right here!

> We might consider using turbo here as well? (as in headless-form)

Totally!  I'll work on adding turbo right after this PR as to unblock @nicolechung with merge conflicts by me moving things around 😂 .